votor-messages/src/consensus_message.rs

```diff
@@ -49,123 +49,60 @@ pub enum Certificate {
+            | Self::Notarize(slot, _)
+            | Self::NotarizeFallback(slot, _)
+            | Self::Skip(slot) => *slot,
         }
     }
 
-    /// Is this a fast finalize certificate?
-    pub fn is_fast_finalization(&self) -> bool {
-        matches!(self, Self::FinalizeFast(_, _))
-    }
-
-    /// Is this a finalize / fast finalize certificate?
-    pub fn is_finalization(&self) -> bool {
-        matches!(self, Self::Finalize(_) | Self::FinalizeFast(_, _))
-    }
-
-    /// Is this a notarize fallback certificate?
-    pub fn is_notarize_fallback(&self) -> bool {
-        matches!(self, Self::NotarizeFallback(_, _))
-    }
-
-    /// Is this a skip certificate?
-    pub fn is_skip(&self) -> bool {
-        matches!(self, Self::Skip(_))
-    }
-
     /// Gets the block associated with this certificate, if present
     pub fn to_block(self) -> Option<Block> {
         match self {
-            Certificate::Finalize(_) | Certificate::Skip(_) => None,
-            Certificate::Notarize(slot, block_id)
-            | Certificate::NotarizeFallback(slot, block_id)
-            | Certificate::FinalizeFast(slot, block_id) => Some((slot, block_id)),
+            Self::Finalize(_) | Self::Skip(_) => None,
+            Self::Notarize(slot, block_id)
+            | Self::NotarizeFallback(slot, block_id)
+            | Self::FinalizeFast(slot, block_id) => Some((slot, block_id)),
         }
     }
 }
 
+/// The actual certificate with the aggregate signature and bitmap for which validators are included in the aggregate.
+/// BLS vote message, we need rank to look up pubkey
 #[cfg_attr(
     feature = "frozen-abi",
     derive(AbiExample),
-    frozen_abi(digest = "4pnAg6WpCFLfDL8w3uujV6k7uqw3RThCLQiD6whTxhbN")
+    frozen_abi(digest = "2jUyAYKXdK7gfncAx3JxhdUfA8DrkVkcbDB6J5tsiuEA")
 )]
 #[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
-/// BLS vote message, we need rank to look up pubkey
-pub struct CertificateMessage {
-    /// The certificate
-    pub certificate: Certificate,
-    /// The signature
+pub struct Certificate {
+    /// The certificate type.
+    pub cert_type: CertificateType,
+    /// The aggregate signature.
     pub signature: BLSSignature,
-    /// The bitmap for validators, see solana-signer-store for encoding format
+    /// The bitmap for validators' signatures are included in the aggregate.
+    /// See solana-signer-store for encoding format.
     pub bitmap: Vec<u8>,
 }
 
+/// A consensus message sent between validators.
 #[cfg_attr(
     feature = "frozen-abi",
     derive(AbiExample, AbiEnumVisitor),
-    frozen_abi(digest = "AusWCevDpUnnKrAjJg7NgsiZZn5pTXaLUghGfPHrsAFA")
+    frozen_abi(digest = "7r7dyUzmnYbxug6r7QkggXgBH5WUWvuC2Z9UcXLJfBgm")
 )]
 #[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
 #[allow(clippy::large_enum_variant)]
-/// BLS message data in Alpenglow
 pub enum ConsensusMessage {
-    /// Vote message, with the vote and the rank of the validator.
+    /// Vote message.
```

Comment:
We can remove the comment now, same below.

Reply:
Err, let's at least add something useful, like vote is from a single party, certificate is aggregate of votes from multiple parties, or something like that.

Comments this simple make next person want to remove it and waste time :)